services/pull/merge.go

```diff
@@ -56,7 +56,7 @@ func getMergeMessage(ctx context.Context, baseGitRepo *git.Repository, pr *issue
 		issueReference = "!"
 	}
 
-	reviewedOn := fmt.Sprintf("Reviewed-on: %s/%s", setting.AppURL, pr.Issue.Link())
+	reviewedOn := fmt.Sprintf("Reviewed-on: %s%s", setting.AppURL, strings.TrimPrefix(pr.Issue.Link(), "/"))
```

Comment:
It would cause duplicate-sub-path bug like #31361

I think the link/path problem should be completed fixed by #31365 https://github.com/go-gitea/gitea/pull/31365/files#diff-a5f6c45ee89a1361d40b8356eae9e116192eabefa574104297019a4af9409e2bL59

Could you review and try it?